scripts/install-deps.sh

```diff
@@ -456,6 +487,7 @@ do_install_do() {
     local do_cmake_options=(
         "-DDO_BUILD_TESTS:BOOL=OFF"
         "-DDO_INCLUDE_SDK=ON"
+        "-DOPENSSL_ROOT_DIR=$work_folder/deviceupdate-openssl"
```

Comment:
Does DO only work with OpenSSL 1.x or can it work with OpenSSL 3.x?
Same question for ABSFUU below.